src/main/java/com/spotifyapi/controller/SpotifyController.java

```diff
@@ -0,0 +1,24 @@
+package com.spotifyapi.controller;
+
+import com.spotifyapi.model.SpotifyArtist;
+import com.spotifyapi.service.SpotifyService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/spotify/")
```

Comment:
you use / in mapping here but not in getMyArtist(). Make it consistent, use one style

Reply:
Done!